var/spack/repos/builtin/packages/matlab/package.py

```diff
@@ -42,8 +42,10 @@ class Matlab(Package):
     mirror, see http://spack.readthedocs.io/en/latest/mirrors.html"""
 
     homepage = "https://www.mathworks.com/products/matlab.html"
+    
```

Comment:
this will make flake8 unhappy

Reply:
In addition to making flake8 unhappy, `matlab` isn't related to blast plus (even though it's just whitespace).